web/pdf_page_view.js

```diff
@@ -63,6 +63,7 @@ import { viewerCompatibilityParams } from "./viewer_compatibility.js";
  *   total pixels, i.e. width * height. Use -1 for no limit. The default value
  *   is 4096 * 4096 (16 mega-pixels).
  * @property {IL10n} l10n - Localization service.
+ * @property {boolean} enableScripting - Enable embedded script execution.
```

Comment:
Please mention the default value here as well:
```javascript
 *   The default value is `false`.
```
`*   The default value is `false`.`